### DISCUSSION TITLE
Google Workspace's SMTP Relay fails every time.

### DISCUSSION THREAD

**Discussion by ethamitc** [pterodactyl/panel#3052] [Support]
**Background (please complete the following information):**
* Panel or Daemon: Panel
* Version of Panel/Daemon: 1.2.2
* Server's OS: Debian 10
* Your Computer's OS & Browser: Windows 10 20H2 / Microsoft Edge

**Describe the bug**
The SMTP mailer inside of Pterodactyl fails to send emails from the Google Worksapce SMTP Relay (smtp-relay.gmail.com).
I've done some googling and it seems to be an issue with swiftmailer, my logs confirms this; `[2021-01-28 18:52:57] production.ERROR: Swift_TransportException: Expected response code 250 but got an empty response in /var/www/pterodactyl/vendor/swiftmailer/swiftmailer/lib/classes/Swift/Transport/AbstractSmtpTransport.php:445`.

**To Reproduce**
1. Setup SMTP to use Google's smtp-relay.gmail.com
2. Create new user or do any action that would trigger an email.
3. It will fail (The only time it hasn't for me is sending to an email inside of the Google Workspace account)

**Expected behavior**
It should send the email without any issues.

**Comment by armeniax** [comment]
I am having the same issue with workspace (former gsuite) email smtp credentials recently.

**Reply by armeniax** [reply]
@ethamitc what cloud environment you are using? we are on google cloud, seems like this might be an issue based on my research. Will add couple of links here to follow.

https://stackoverflow.com/questions/65885428/laravel-smtp-sending-email-by-queue-receiving-empty-response
https://community.nodebb.org/topic/15310/email-suddenly-no-longer-works-with-smtp-relay-google-workspace/6?lang=en-US

**Reply by ethamitc** [reply]
I'm not using google cloud. It's a standard VM running on Proxmox.

**Reply by armeniax** [reply]
@ethamitc 

Google support rep advised to use ```smtp.gmail.com``` instead of ```smtp-relay.gmail.com``` which worked for me.
On Monday they will most probably announce about the issue.

The problem is on their side, they know about it, many people having this issue.

**Reply by ethamitc** [reply]
Yeah I sat with support for almsot 2 hours. The only issue with `smtp.gmail.com` is that it sends as the smtp user, which is no good for my use case.

**Reply by ethamitc** [reply]
> @ethamitc
> 
> Google support rep advised to use `smtp.gmail.com` instead of `smtp-relay.gmail.com` which worked for me.
> On Monday they will most probably announce about the issue.
> 
> The problem is on their side, they know about it, many people having this issue.

Hi @armeniax, just as an update from what I can tell is the SMTP Relay is meant to be internal only and only allows you to send emails to emails within your Google Workspaces org.

**Comment by MjeTjeu** [comment]
**Maybe this helps:**

[Laravel-incorrectly-handling-ehlo-for-smtp](https://laracasts.com/discuss/channels/laravel/laravel-incorrectly-handling-ehlo-for-smtp-from-artisancli-issues-with-gmails-smtp-relay-resolved)
